src/ontology/cl-edit.owl

```diff
@@ -26956,6 +26963,7 @@ SubClassOf(obo:CL_1000803 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000521
 
 # Class: obo:CL_1000804 (kidney outer medulla interstitial cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:21359760") Annotation(oboInOwl:hasDbXref "PMID:8694267") obo:IAO_0000115 obo:CL_1000804 "A kidney cell that is part of an interstitial compartment of an outer renal medulla.")
```

Comment:
Consider removing PMID:[25266210](https://pubmed.ncbi.nlm.nih.gov/25266210), since it is not very specific for this term.

Reply:
I meant PMID:21359760, I probably didn't copy and paste a past copied ref.